frontend/tests/common/constants.ts

```diff
@@ -26,3 +26,14 @@ export const BLUEPRINT_SAFE_TYPE_OPTIONS = { delay: 50 };
 export const SKIP_LOGIN = process.env.SKIP_LOGIN === "true" || false;
 
 export const LOGIN_STATE_FILENAME = "loginState.json";
+
+export const SHARED_LINK =
```

Comment:
let's use `TEST_URL`, otherwise the tests will fail in envs other than `localhost`, such ss dev/staging/prod/rdev! I thought this was fixed in a different PR, not sure how it came back